ietf/draft-ietf-jmap-calendars-18.txt

```diff
@@ -0,0 +1,3360 @@
+
+Internet-Draft               JMAP Calendars                    June 2024
+
+
+   +----------------------------------------+
+   |                                        |
+   |  Reminder: Team catchup                |
+   |  Today at 10am (in 1 hour)             |
+   |                             [Snooze\/] |
+   +----------------------------------------+
+
+   The user snoozes the notification for 30 minutes.  The client
+   dismisses the current notification and sends an update to the event
+   to the server:
+
+   NOTE: '\' line wrapping per RFC 8792
+
+   [
+     ["CalendarEvent/set", {
+           "accountId": "a0x9",
+           "update": {
+         "E-01c9626e-1490-43df-a34f-457021256281": {
+           "alerts/7519a951-1e6f-4a6c-b08b-20dd2e5a89cd\
+                   /acknowledged": "20230110T23:00:31Z",
+           "alerts/86b0-318b8291045f": {
+             "@type": "Alert",
+             "action": "display",
+             "trigger": {
+               "@type": "AbsoluteTrigger",
+               "when": "20230210T23:30:00Z",
+               "relatedTo": {
+                 "7519a951-1e6f-4a6c-b08b-20dd2e5a89cd": {
+                   "@type": "Relation",
+                   "relation": {
+                     "parent": true
+                   }
+                 }
+                   }
+             }
+           }
+         }
+       }
+     }, "0"]
+   ]
+
+   Any other connected client will receive a push, sync the change and
+   dismiss any duplicate alert.  After the snooze time has elapsed, the
+   new alert will trigger.
+
+
+
+
+
+
+
+Jenkins & Douglass       Expires 9 December 2024               [Page 47]
+
+Internet-Draft               JMAP Calendars                    June 2024
+
+
+8.4.  Changing the default calendar
+
+   The client tries to change the default calendar from "Work" to
+   "Private" (and makes no other change):
+
+   [
+     ["Calendar/set", {
+       "accountId": "a0x9",
+       "onSuccessSetIsDefault": "062adcfa-105d-455c-bc60-6db68b69c3f3"
+     }, "0"]
+   ]
+
+   The server allows the change, returning the following response:
+
+   [
+     ["Calendar/set", {
+       "accountId": "a0x9",
+       "updated": {
+         "062adcfa-105d-455c-bc60-6db68b69c3f3": {
+           "isDefault": true
+         },
+         "3ddf2ad7-0e0c-4fb5-852d-f0ff56f3c662": {
+           "isDefault": false
+         }
+       }
+     }, "0"]
+   ]
+
+8.5.  Parsing an iCalendar file
+
+   The client makes a request to parse the calendar event from a blob id
+   representing an icalendar file:
+
+   [
+     [ "CalendarEvent/parse", {
+       "accountId": "a0x9",
+       "blobIds": ["Ge682d5d7aad50b3a4f7180a7ed9276476485ea52"]
+     }, "c1"]
+   ]
+
+   The server responds:
+
+
+
+
+
+
+
+
+
+
+Jenkins & Douglass       Expires 9 December 2024               [Page 48]
+
+Internet-Draft               JMAP Calendars                    June 2024
+
+
+ [[ "CalendarEvent/parse", {
+   "accountId": "ue150411c",
```

Comment:
This accountId does not match `"a0x9"` above

Reply:
Thanks, fixed.